src/watch/watch.ts

```diff
@@ -43,16 +62,25 @@ export class Watcher {
 		for (const task of this.tasks) {
 			task.close();
 		}
+		this.emitter.emit('close');
 		this.emitter.removeAllListeners();
 	}
 
-	emit(event: string, value?: any) {
```

Comment:
I removed method, since it just proxy, use `emitter.emit` isn't harder + have better type-checking.